### PR TITLE
add CTAs to in-app apm setup docs

### PR DESCRIPTION
**Pull request by andrewardito**
### What does this PR do?
Adds CTAs for in app onboarding

### Preview link
https://docs-staging.datadoghq.com/andrew.ardito/addinappcallouts/

### CODE REVIEW

File: content/en/getting_started/tracing/_index.md [hunk 1]
```diff
@@ -53,9 +53,18 @@ After a few minutes, verify the Agent is connected to your account by checking t
 
 ## Datadog APM
 
+### Follow the in-app Documentation (Recommended)
+
+For the remaining steps, it is recommended to follow the [Quickstart instructions][10] within the Datadog app for the best experience, including:
```
Comment (kayayarai) [inline content/en/getting_started/tracing/_index.md:58]:
```suggestion
For the remaining steps, follow the [Quickstart instructions][10] within the Datadog app for the best experience, including:
```

File: content/en/getting_started/tracing/_index.md [hunk 2]
```diff
@@ -53,9 +53,18 @@ After a few minutes, verify the Agent is connected to your account by checking t
 
 ## Datadog APM
 
+### Follow the in-app Documentation (Recommended)
```
Comment (kayayarai) [inline content/en/getting_started/tracing/_index.md:56]:
```suggestion
### Follow the in-app documentation (Recommended)
```

File: content/en/getting_started/tracing/_index.md [hunk 3]
```diff
@@ -53,9 +53,18 @@ After a few minutes, verify the Agent is connected to your account by checking t
...
+
+For the remaining steps, it is recommended to follow the [Quickstart instructions][10] within the Datadog app for the best experience, including:
+
+- Step-by-step instructions scoped to your deployment configuration (in this case, a host-based deployment).
+- Dynamically set `service`, `env` and `version` tags.
```
Comment (kayayarai) [inline content/en/getting_started/tracing/_index.md:61]:
```suggestion
- Dynamically set `service`, `env`, and `version` tags.
```

File: content/en/getting_started/tracing/_index.md [hunk 4]
```diff
@@ -78,12 +87,20 @@ And in `trace-agent.log`:
...
+Alternatively, name your environment by updating `datadog.yaml` to set `env` under `apm_config`. To learn more about setting `env` for APM, see the [setting primary tags to scope guide][13].
 
 ## APM application
 
+### Follow the in-app Documentation (Recommended)
+
+Follow the [Quickstart instructions][10] within the Datadog app for the best experience, including:
+
+- Step-by-step instructions scoped to your deployment configuration (hosts, Docker, Kubernetes, or Amazon ECS).
+- Dynamically set `service`, `env` and `version` tags.
```
Comment (kayayarai) [inline content/en/getting_started/tracing/_index.md:101]:
```suggestion
- Dynamically set `service`, `env`, and `version` tags.
```

File: content/en/getting_started/tracing/_index.md [hunk 5]
```diff
@@ -78,12 +87,20 @@ And in `trace-agent.log`:
...
+- Enable App Analytics, and Trace ID injection into logs during setup.
```
Comment (kayayarai) [inline content/en/getting_started/tracing/_index.md:102]:
```suggestion
- Enable App Analytics and Trace ID injection into logs during setup.
```

File: content/en/getting_started/tracing/_index.md [hunk 6]
```diff
@@ -78,12 +87,20 @@ And in `trace-agent.log`:
...
 Additionally, if your tracer has logs injection enabled then the `env` will be consistent across traces and logs.
-Read more about how this works in [Unified Service Tagging][11].
+Read more about how this works in [Unified Service Tagging][12].
 
-Alternatively, name your environment by updating `datadog.yaml` to set `env` under `apm_config`. To learn more about setting `env` for APM, see the [setting primary tags to scope guide][12].
+Alternatively, name your environment by updating `datadog.yaml` to set `env` under `apm_config`. To learn more about setting `env` for APM, see the [setting primary tags to scope guide][13].
 
 ## APM application
 
+### Follow the in-app Documentation (Recommended)
```
Comment (kayayarai) [inline content/en/getting_started/tracing/_index.md:96]:
```suggestion
### Follow the in-app documentation (Recommended)
```

File: content/en/tracing/send_traces/_index.md [hunk 7]
```diff
@@ -35,6 +35,10 @@ To get an overview of all the possible settings for APM, take a look at the Agen
 
 See the specific setup instructions to ensure that the Agent is configured to receive traces in a containerized environment:
 
+### Follow the in-app Documentation (Recommended)
```
Comment (kayayarai) [inline content/en/tracing/send_traces/_index.md:38]:
```suggestion
### Follow the in-app documentation (Recommended)
```

File: content/en/tracing/setup/dotnet-core.md [hunk 8]
```diff
@@ -27,7 +27,13 @@ The .NET Tracer supports automatic instrumentation on .NET Core 2.1 and 3.1, as
 
 ## Getting Started
 
-If you already have a Datadog account you can find [step-by-step instructions][2] in our in-app guides for either host-based or container-based set ups.
+### Follow the in-app Documentation (Recommended)
```
Comment (kayayarai) [inline content/en/tracing/setup/dotnet-core.md:30]:
```suggestion
### Follow the in-app documentation (Recommended)
```

File: content/en/tracing/setup/dotnet-core.md [hunk 9]
```diff
@@ -27,7 +27,13 @@ The .NET Tracer supports automatic instrumentation on .NET Core 2.1 and 3.1, as
 
 ## Getting Started
```
Comment (kayayarai) [inline content/en/tracing/setup/dotnet-core.md:28]:
```suggestion
## Getting started
```

File: content/en/tracing/setup/dotnet-framework.md [hunk 10]
```diff
@@ -30,7 +30,13 @@ The .NET Tracer supports automatic instrumentation on .NET Framework 4.5 and abo
...
 ## Getting Started
 
-If you already have a Datadog account you can find [step-by-step instructions][2] in our in-app guides for either host-based or container-based set ups.
+### Follow the in-app Documentation (Recommended)
+
+Follow the [Quickstart instructions][2] within the Datadog app for the best experience, including:
+
+- Step-by-step instructions scoped to your deployment configuration (hosts, Docker, Kubernetes, or Amazon ECS).
+- Dynamically set `service`, `env` and `version` tags.
+- Enable App Analytics, and Trace ID injection into logs during setup.
```
Comment (kayayarai) [inline content/en/tracing/setup/dotnet-framework.md:39]: lowercase started, lowercase documentation, add comma after `env`, remove comma after Analytics

File: content/en/tracing/setup/go.md [hunk 11]
```diff
@@ -34,9 +34,15 @@ Consult the [migration document][5] if you need to migrate from an older version
 
 ### Installation
 
-If you already have a Datadog account you can find [step-by-step instructions][6] in our in-app guides for either host-based or container-based set ups.
+#### Follow the in-app Documentation (Recommended)
```
Comment (kayayarai) [inline content/en/tracing/setup/go.md:37]:
```suggestion
#### Follow the in-app documentation (Recommended)
```

File: content/en/tracing/setup/go.md [hunk 12]
```diff
@@ -34,9 +34,15 @@ Consult the [migration document][5] if you need to migrate from an older version
...
 
-First [install and configure the Datadog Agent][7]. See the additional documentation for [tracing Docker applications][8] or [Kubernetes applications][9].
+Follow the [Quickstart instructions][6] within the Datadog app for the best experience, including:
+
+- Step-by-step instructions scoped to your deployment configuration (hosts, Docker, Kubernetes, or Amazon ECS).
+- Dynamically set `service`, `env` and `version` tags.
```
Comment (kayayarai) [inline content/en/tracing/setup/go.md:42]:
```suggestion
- Dynamically set `service`, `env`, and `version` tags.
```

File: content/en/tracing/setup/nodejs.md [hunk 13]
```diff
@@ -27,7 +27,13 @@ The NodeJS Tracer officially supports versions `>=8`. Only even versions like 8.
...
 ## Installation And Getting Started
 
-If you already have a Datadog account you can find [step-by-step instructions][2] in our in-app guides for either host-based or container-based set ups.
+### Follow the in-app Documentation (Recommended)
+
+Follow the [Quickstart instructions][2] within the Datadog app for the best experience, including:
+
+- Step-by-step instructions scoped to your deployment configuration (hosts, Docker, Kubernetes, or Amazon ECS).
+- Dynamically set `service`, `env` and `version` tags.
+- Enable App Analytics, and Trace ID injection into logs during setup.
```
Comment (kayayarai) [inline content/en/tracing/setup/nodejs.md:36]: lowercase getting started, lowercase documentation, add comma after `env`, remove comma after Analytics

File: content/en/tracing/setup/php.md [hunk 14]
```diff
@@ -25,7 +25,13 @@ For a full list of supported libraries and language versions, visit the [Compati
...
 ## Installation and Getting Started
 
-If you already have a Datadog account you can find [step-by-step instructions][2] in our in-app guides for either host-based or container-based set ups.
+### Follow the in-app Documentation (Recommended)
+
+Follow the [Quickstart instructions][2] within the Datadog app for the best experience, including:
+
+- Step-by-step instructions scoped to your deployment configuration (hosts, Docker, Kubernetes, or Amazon ECS).
+- Dynamically set `service`, `env` and `version` tags.
+- Enable App Analytics during setup.
```
Comment (kayayarai) [inline content/en/tracing/setup/php.md:34]: lowercase started, lowercase documentation, add comma after `env`

File: content/en/tracing/setup/python.md [hunk 15]
```diff
@@ -25,7 +25,13 @@ Python versions `2.7+` and `3.5+` are supported.  For a full list of supported l
...
 ## Installation and Getting Started
 
-If you already have a Datadog account you can find [step-by-step instructions][2] in our in-app guides for either host-based or container-based set ups.
+### Follow the in-app Documentation (Recommended)
+
+Follow the [Quickstart instructions][2] within the Datadog app for the best experience, including:
+
+- Step-by-step instructions scoped to your deployment configuration (hosts, Docker, Kubernetes, or Amazon ECS).
+- Dynamically set `service`, `env` and `version` tags.
+- Enable the Continuous Profiler, App Analytics, and Trace ID injection into logs during setup.
```
Comment (kayayarai) [inline content/en/tracing/setup/python.md:34]: lowercase getting started, lowercase documentation, add comma after `env`

File: content/en/tracing/setup/nodejs.md [hunk 16]
```diff
@@ -27,7 +27,13 @@ The NodeJS Tracer officially supports versions `>=8`. Only even versions like 8.
 
 ## Installation And Getting Started
 
-If you already have a Datadog account you can find [step-by-step instructions][2] in our in-app guides for either host-based or container-based set ups.
+### Follow the in-app Documentation (Recommended)
+
+Follow the [Quickstart instructions][2] within the Datadog app for the best experience, including:
+
+- Step-by-step instructions scoped to your deployment configuration (hosts, Docker, Kubernetes, or Amazon ECS).
+- Dynamically set `service`, `env` and `version` tags.
```
Comment (andrewardito) [inline content/en/tracing/setup/nodejs.md:35]:
```suggestion
- Dynamically set `service`, `env`, and `version` tags.
```

File: content/en/tracing/setup/nodejs.md (hunk 13 above)
Comment (andrewardito) [inline content/en/tracing/setup/nodejs.md:36]:
```suggestion
- Enable App Analytics and Trace ID injection into logs during setup.
```

File: content/en/tracing/setup/php.md [hunk 17]
```diff
@@ -25,7 +25,13 @@ For a full list of supported libraries and language versions, visit the [Compati
 
 ## Installation and Getting Started
 
-If you already have a Datadog account you can find [step-by-step instructions][2] in our in-app guides for either host-based or container-based set ups.
+### Follow the in-app Documentation (Recommended)
+
+Follow the [Quickstart instructions][2] within the Datadog app for the best experience, including:
+
+- Step-by-step instructions scoped to your deployment configuration (hosts, Docker, Kubernetes, or Amazon ECS).
+- Dynamically set `service`, `env` and `version` tags.
```
Comment (andrewardito) [inline content/en/tracing/setup/php.md:33]:
```suggestion
- Dynamically set `service`, `env`, and `version` tags.
```

File: content/en/tracing/setup/php.md [hunk 18]
```diff
@@ -25,7 +25,13 @@ For a full list of supported libraries and language versions, visit the [Compati
 
 ## Installation and Getting Started
 
-If you already have a Datadog account you can find [step-by-step instructions][2] in our in-app guides for either host-based or container-based set ups.
+### Follow the in-app Documentation (Recommended)
```
Comment (andrewardito) [inline content/en/tracing/setup/php.md:28]:
```suggestion
### Follow the in-app documentation (Recommended)
```

File: content/en/tracing/setup/dotnet-framework.md [hunk 19]
```diff
@@ -30,7 +30,13 @@ The .NET Tracer supports automatic instrumentation on .NET Framework 4.5 and abo
 
 ## Getting Started
 
-If you already have a Datadog account you can find [step-by-step instructions][2] in our in-app guides for either host-based or container-based set ups.
+### Follow the in-app Documentation (Recommended)
```
Comment (andrewardito) [inline content/en/tracing/setup/dotnet-framework.md:33]:
```suggestion
### Follow the in-app documentation (Recommended)
```

File: content/en/tracing/setup/dotnet-framework.md [hunk 20]
```diff
@@ -30,7 +30,13 @@ The .NET Tracer supports automatic instrumentation on .NET Framework 4.5 and abo
...
+
+Follow the [Quickstart instructions][2] within the Datadog app for the best experience, including:
+
+- Step-by-step instructions scoped to your deployment configuration (hosts, Docker, Kubernetes, or Amazon ECS).
+- Dynamically set `service`, `env` and `version` tags.
```
Comment (andrewardito) [inline content/en/tracing/setup/dotnet-framework.md:38]:
```suggestion
- Dynamically set `service`, `env`, and `version` tags.
```

File: content/en/tracing/setup/dotnet-framework.md (hunk 10 above)
Comment (andrewardito) [inline content/en/tracing/setup/dotnet-framework.md:39]:
```suggestion
- Enable App Analytics and Trace ID injection into logs during setup.
```

File: content/en/tracing/setup/python.md [hunk 21]
```diff
@@ -25,7 +25,13 @@ Python versions `2.7+` and `3.5+` are supported.  For a full list of supported l
 
 ## Installation and Getting Started
 
-If you already have a Datadog account you can find [step-by-step instructions][2] in our in-app guides for either host-based or container-based set ups.
+### Follow the in-app Documentation (Recommended)
```
Comment (andrewardito) [inline content/en/tracing/setup/python.md:28]:
```suggestion
### Follow the in-app documentation (Recommended)
```

File: content/en/tracing/setup/python.md [hunk 22]
```diff
@@ -25,7 +25,13 @@ Python versions `2.7+` and `3.5+` are supported.  For a full list of supported l
...
+
+Follow the [Quickstart instructions][2] within the Datadog app for the best experience, including:
+
+- Step-by-step instructions scoped to your deployment configuration (hosts, Docker, Kubernetes, or Amazon ECS).
+- Dynamically set `service`, `env` and `version` tags.
```
Comment (andrewardito) [inline content/en/tracing/setup/python.md:33]:
```suggestion
- Dynamically set `service`, `env`, and `version` tags.
```

File: content/en/tracing/setup/nodejs.md [hunk 23]
```diff
@@ -27,7 +27,13 @@ The NodeJS Tracer officially supports versions `>=8`. Only even versions like 8.
 
 ## Installation And Getting Started
 
-If you already have a Datadog account you can find [step-by-step instructions][2] in our in-app guides for either host-based or container-based set ups.
+### Follow the in-app Documentation (Recommended)
```
Comment (andrewardito) [inline content/en/tracing/setup/nodejs.md:30]:
```suggestion
### Follow the in-app documentation (Recommended)
```

File: content/en/tracing/setup/dotnet-framework.md (hunk 10 above)
Comment (andrewardito) [inline content/en/tracing/setup/dotnet-framework.md:39]:
```suggestion
- Enable App Analytics and Trace ID injection into logs during setup.
```

File: content/en/tracing/setup/dotnet-core.md [hunk 24]
```diff
@@ -27,7 +27,13 @@ The .NET Tracer supports automatic instrumentation on .NET Core 2.1 and 3.1, as
...
 
-If you already have a Datadog account you can find [step-by-step instructions][2] in our in-app guides for either host-based or container-based set ups.
+### Follow the in-app Documentation (Recommended)
+
+Follow the [Quickstart instructions][2] within the Datadog app for the best experience, including:
+
+- Step-by-step instructions scoped to your deployment configuration (hosts, Docker, Kubernetes, or Amazon ECS).
+- Dynamically set `service`, `env` and `version` tags.
```
Comment (andrewardito) [inline content/en/tracing/setup/dotnet-core.md:35]:
```suggestion
- Dynamically set `service`, `env`, and `version` tags.
```

File: content/en/tracing/setup/dotnet-core.md [hunk 25]
```diff
@@ -27,7 +27,13 @@ The .NET Tracer supports automatic instrumentation on .NET Core 2.1 and 3.1, as
...
+- Enable App Analytics, and Trace ID injection into logs during setup.
```
Comment (andrewardito) [inline content/en/tracing/setup/dotnet-core.md:36]:
```suggestion
- Enable App Analytics and Trace ID injection into logs during setup.
```

File: content/en/tracing/setup/python.md [hunk 26]
```diff
@@ -25,7 +25,13 @@ Python versions `2.7+` and `3.5+` are supported.  For a full list of supported l
 
 ## Installation and Getting Started
```
Comment (andrewardito) [inline content/en/tracing/setup/python.md:26]:
```suggestion
## Installation and getting Started
```

File: content/en/tracing/setup/php.md [hunk 27]
```diff
@@ -25,7 +25,13 @@ For a full list of supported libraries and language versions, visit the [Compati
 
 ## Installation and Getting Started
```
Comment (andrewardito) [inline content/en/tracing/setup/php.md:26]:
```suggestion
## Installation and Getting started
```

File: content/en/tracing/setup/nodejs.md [hunk 28]
```diff
@@ -27,7 +27,13 @@ The NodeJS Tracer officially supports versions `>=8`. Only even versions like 8.
 
 ## Installation And Getting Started
```
Comment (andrewardito) [inline content/en/tracing/setup/nodejs.md:28]:
```suggestion
## Installation And getting started
```

File: content/en/tracing/setup/dotnet-framework.md [hunk 29]
```diff
@@ -30,7 +30,13 @@ The .NET Tracer supports automatic instrumentation on .NET Framework 4.5 and abo
 
 ## Getting Started
```
Comment (andrewardito) [inline content/en/tracing/setup/dotnet-framework.md:31]:
```suggestion
## Getting started
```

File: content/en/tracing/setup/nodejs.md [hunk 30]
```diff
@@ -25,9 +25,15 @@ further_reading:
 
 The NodeJS Tracer officially supports versions `>=8`. Only even versions like 8.x and 10.x are officially supported. Odd versions like 9.x and 11.x should work but are not officially supported. For a full list of supported libraries, visit the [Compatibility Requirements][1] page.
 
-## Installation And Getting Started
+## Installation And getting started
```
Comment (andrewardito) [inline content/en/tracing/setup/nodejs.md:28]:
```suggestion
## Installation and getting started
```

File: content/en/tracing/setup/python.md [hunk 31]
```diff
@@ -23,9 +23,15 @@ further_reading:
 
 Python versions `2.7+` and `3.5+` are supported.  For a full list of supported libraries, visit the [Compatibility Requirements][1] page.
 
-## Installation and Getting Started
+## Installation and getting Started
```
Comment (andrewardito) [inline content/en/tracing/setup/python.md:26]:
```suggestion
## Installation and getting started
```

**Pull request merged**